api/Document.json

```diff
@@ -7164,12 +7210,24 @@
               "version_added": "11",
               "alternative_name": "onmsfullscreenchange"
             },
-            "opera": {
-              "version_added": true
-            },
-            "opera_android": {
-              "version_added": true
-            },
+            "opera": [
+              {
+                "version_added": "58"
+              },
+              {
+                "version_added": "32",
```

Comment:
The data that this is mirroring is wrong. `document.onwebkitfullscreenchange` was in Chrome 15 per http://mdn-bcd-collector.appspot.com/tests/api/Document/onwebkitfullscreenchange, probably earlier. Can you update the Chrome data for that, or just not mirror the incorrect data until it's fixed?